src/lib/libdll/dlldefs.h

```diff
@@ -0,0 +1,170 @@
+/***********************************************************************
+*                                                                      *
+*               This software is part of the ast package               *
+*          Copyright (c) 1997-2018 AT&T Intellectual Property          *
+*                      and is licensed under the                       *
+*                 Eclipse Public License, Version 1.0                  *
+*                    by AT&T Intellectual Property                     *
+*                                                                      *
+*                A copy of the License is available at                 *
+*          http://www.eclipse.org/org/documents/epl-v10.html           *
+*         (with md5 checksum b35adb5213ca9657e911e9befb180842)         *
+*                                                                      *
+*              Information and Software Systems Research               *
+*                            AT&T Research                             *
+*                           Florham Park NJ                            *
+*                                                                      *
+*               Glenn Fowler <glenn.s.fowler@gmail.com>                *
+*                                                                      *
+***********************************************************************/
+
+/* : : generated by proto : : */
+/* : : generated from /home/situ/test/ast/src/lib/libdll/features/dll by iffe version 2013-11-14 : : */
+                  
+#ifndef _def_dll_dll
+#if !defined(__PROTO__)
```

Comment:
What happens if you remove everything from here to line 81? AFAICT this is just boilerplate for dealing with pre-ANSI C compilers.